src/transformers/onnx/features.py

```diff
@@ -457,6 +457,9 @@ class FeaturesManager:
             "object-detection",
             onnx_config_cls="models.yolos.YolosOnnxConfig",
         ),
+        "swin": supported_features_mapping(
```

Comment:
Please order this entry alphabetically, i.e. just before `t5` (note to self: should automate this)

Reply:
@lewtun I did that but : https://app.circleci.com/pipelines/github/huggingface/transformers/46997/workflows/627f2d41-0b49-4f91-bf83-da5c5db701e8/jobs/555035/parallel-runs/0/steps/0-111

is showing a styling error in other files, should I run `make style` on other files? and there's a merge conflict tests/onnx/test_onnx_v2.py